docker-compose.override.yml

```diff
@@ -4,7 +4,7 @@ services:
   app:
     build:
       dockerfile: ./Dockerfile-development
-    command: sh -c "yarn install --frozen-lockfile && ./scripts/wait-for-it.sh postgres:5432 -s -t 40 -- npx pubsweet server"
```

Comment:
I don't know what the `yarn` command does but I think it was necessary in local use to apply local changes to `server/` and similar on top of the Docker image?